orchestrator/settings.py

```diff
@@ -50,6 +50,8 @@ class AppSettings(BaseSettings):
     CACHE_HOST: str = "127.0.0.1"
     CACHE_PORT: int = 6379
     CACHE_DOMAIN_MODELS: bool = False
+    # The key can be any length. However, the recommended size is 1024 bits.
+    CACHE_HMAC_SECRET: str = "wl9WoQju0c"  # HMAC signing key, used when pickling results in the cache
```

Comment:
Not sure if a default secret serves any purpose. 

I think it should default to None (along with an explanation in the Readme that a secret should be injected in the ENV at runtime)

Reply:
Changed it. And added some info to the README.